docker/Dockerfile.api

```diff
@@ -0,0 +1,19 @@
+FROM node:11-alpine AS build
+
+COPY package*.json ./
+RUN npm install --production
+
+FROM node:11-alpine
+
+RUN npm install -g gulp gulp-cli
+
+WORKDIR /app
+
+VOLUME /app/prebid.js
+
+COPY --from=build ./node_modules ./node_modules
+COPY . .
+
+EXPOSE 3000
```

Comment:
how will this map to port 80? Will we continue to run apache in front?

Reply:
Port 80 will be exposed on the Application Load Balancer and it will proxy requests to the API running on port 3000 on EC2 instances.